drivers/counter/counter_ite_it8xxx2.c

```diff
@@ -283,7 +283,7 @@ static int counter_it8xxx2_init(const struct device *dev)
 	uint8_t et8_ctrl = counter_it8xxx2_read8(dev, ET8CTRL);
 
 	LOG_DBG("max top value = 0x%08x", config->info.max_top_value);
```

Comment:
Cast to `(uint32_t)`?
If several counter drivers are enabled and 1 sets `CONFIG_COUNTER_64BITS=y`, then all these ticks related are 64bit in which case `%x` may taint compilers.